src/inc/til.h

```diff
@@ -18,6 +18,111 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
 #pragma warning(suppress : 26446) // Suppress bounds.4 check for subscript operator.
         return cont[i];
     }
+
+    // color is a universal integral 8bpp RGBA (0-255) color type implicitly convertible to/from
+    // a number of other color types.
+#pragma warning(push)
+    // we can't depend on GSL here (some libraries use BLOCK_GSL), so we use static_cast for explicit narrowing
+#pragma warning(disable : 26472)
+    struct color
+    {
+        uint8_t r, g, b, a;

```

Comment:
Should we have a `static_assert` on the size of this type, like `TextColor` does?
```c++
static_assert(sizeof(TextColor) <= 4 * sizeof(BYTE), "We should only need 4B for an entire TextColor. Any more than that is just waste");

```

Reply:
I'm not convinced this is necessary... we'd be testing an invariant, and people shouldn't be changing the representation of til::color without a solidly good reason